src/lib/guesstimator/formatter/formatters/lib.js

```diff
@@ -40,7 +40,7 @@ export const normalTextMixin = Object.assign(
   {}, textMixin,
   {
     errors(g) { return this._normalTextErrors(g.text) },
-    guesstimateType(g) {
+    guesstimateType(g, defaultValue) {
```

Comment:
Good idea; I considered that, but incorrectly thought this was imported in more than just the CI selectable things. Will port this over.